tests/PaymentMathTest.php

```diff
@@ -66,9 +67,9 @@ public function testPrecisionFloat()
         $this->assertEquals('99.90', PaymentMath::subtract('100.00', '0.1'));
         $this->assertEquals('0.49', PaymentMath::add('0.273', '0.226'));
 
-        Config::modify()->set(PaymentMath::class, 'precision', 15);
-        $this->assertEquals('99.900000000000000', PaymentMath::subtract('100.00', '0.1'));
-        $this->assertEquals('0.499000000000000', PaymentMath::add('0.273', '0.226'));
+        Config::modify()->set(PaymentMath::class, 'precision', 12);
```

Comment:
Precision reduced here because of this failure: https://app.travis-ci.com/github/silverstripe/silverstripe-omnipay/jobs/546510897#L1466-L1472. Hence also the recommendation that we should investigate using a battle-tested library instead of `PaymentMath`, or at the very least _require_ the bcmath extension